### ISSUE TITLE
Support of text field with one char like " or \" is breaking the parser.

### ISSUE THREAD

**Issue by bru-no** [mafintosh/csv-parser#44]
I am using command line csv-parser and the csv file that I am processing is not converted in full, it is stopping in the middle, by looking at the fields, one has normal double quote in (") embedded in the text area (\" exactly), I have removed it for test purpose, and now csv-parser goes through all records. This cannot be the final solution for me, as I have other records like this.

I created the below test file which is showing the same issue.

test.csv:

```
"Field1","Field2","Field3","Field4"
"UN","DEUX","TROIS","QUATRE"
"ONE","T\"WO","THREE","FOUR"
"ABC","DEF","GHI","JKL"
```

```
c:>csv-parser test.csv
{"Field1":"UN","Field2":"DEUX","Field3":"TROIS","Field4":"QUATRE"}
{"Field1":"ONE","Field2":"T\\\"WO","Field3":"THREE","Field4":"FOUR\"\r\n\"ABC"}
```

We are missing the 3rd record, something happened with record 2 and 3 been merged, see ABC is inside record 2.

thanks.

**Comment by bru-no** [comment]
The data is coming from microsoft sharepoint list, not from excel, there is probably something around this making the .csv to be different than expected.

**Comment by maxogden** [comment]
as per the csv spec technically you are supposed to escape double quotes like this:

```
"Field1","Field2","Field3","Field4"
"UN","DEUX","TROIS","QUATRE"
"ONE","T""WO","THREE","FOUR"
"ABC","DEF","GHI","JKL"
```

**Comment by bru-no** [comment]
Thanks, I have resolved my problem. 

it is turning... that I started with a .csv file that is ok, next I converted to .json, next I read the fields and created another .csv file (doing some filtering), and next I converted again to .json.

The \" is coming from the first conversion to .json, details is below if you are interested.

SharePoint     field is                      ECI...("Option)
Step 1            extract to .csv          "ECI...(""Option)"          " is converted to double quote ""
Step 2            convert to .json        "ECI...(\"Option)"          it is now converted to \"
Step 3            create new.csv        "ECI...(\"Option)"          my bad, I kept as is
Step 4            convert to .json        problem...                    

so for now, I decided to removed \" globally after step 2 as this is fine for what I am doing at the moment.

thanks for the help.
